app/sfr-search-api/lib/v3Search.js

```diff
@@ -188,7 +188,7 @@ class V3Search {
    * @returns {array} Array of instance objects from the database
    */
   async getInstances(instanceIds) {
-    return this.dbConn.loadInstances(instanceIds, 1)
+    return this.dbConn.loadInstances(instanceIds, 1, ['agents', 'items', 'languages', 'covers', 'dates', 'rights'])
```

Comment:
I'd assign this to a constant somewhere easily editable and commented.  (Should this always be the same value as in `v3Work.js` below?

Reply:
A good idea, and yeah it should be the same as the work detail. There are probably other constants that could be refactored in that way too.